docs/kratos/social-signin/82_patreon.mdx

```diff
@@ -0,0 +1,95 @@
+---
+id: patreon
+title: Add patreon as a social sign-in provider in Ory
+sidebar_label: Patreon
+---
+
+# Patreon
+
+Follow these steps to add Patreon as a social sign-in provider to your project using the Ory CLI:
+
+1. Create a client on the [Patreon Portal](https://www.patreon.com/portal/registration/register-clients).
+2. Ensure you configure the redirect URIs to:
+
+   ```shell
+   https://{project.slug}.projects.oryapis.com/self-service/methods/oidc/callback/patreon
+   ```
+
+3. Create a Jsonnet code snippet to map the desired claims to the Ory Identity schema.
+
+   **Available Claims**  
+   All claims require the `identity` scope
+
+   - Subject: Id of the patreon user
+   - Email: The patron's email (Additionally requires the `identity[email]` scope)
+   - Name: The patron's full name
+   - GivenName: The patron's first name
+   - FamilyName: The patron's last name (Same as `LastName`)
+   - LastName: The patron's last name (Same as `FamilyName`)
+   - Picture: A URL to a patron's profile image if present
+
+   [More information available about patreon's oauth here](https://docs.patreon.com/#apiv2-oauth)
```

Comment:
These are mapped from an API request sent to patreon's user endpoint to get their information. It's not claims from them

Reply:
Sorry, I don't think I understand. However, my point is that I'd rather see an admonition with a link to Patreon docs that give guidance on this matter rather than maintain a bunch of Patreon-specific information in Ory docs ;)

Reply:
I will use an admonition and link to their docs :)
However what I meant by the claims are mapped from an api call is that [Kratos sends sends out an API call and maps the response into claims](https://github.com/ory/kratos/blob/44556a468ef233b18fd0f16a83a4e1b2e5f05dcf/selfservice/strategy/oidc/provider_patreon.go#L111-L120) It is code in the kratos code base hence why I listed the claims.

If you are sure you want me to remove it and link to patreon docs I can do that, I'm just wanting to try get us on the same page before making the change

Reply:
Ok, then go ahead and list the claims here. Just make sure to adapt a nicer format - just like we discussed, an admonition (maybe of the `:::tip` kind?) would fit here. 

Also, please check out our recommendations for using formatting before you start working on this: https://github.com/ory/docs#bolding-code-formatting